doc/ip-interface/loopback-action/rif_loopback_action_testplan.md

```diff
@@ -0,0 +1,151 @@
+config interface loopback-action Vlan100 forward
+config interface loopback-action PortChannel1 drop
+config interface loopback-action Ethernet0.10 drop
+config interface loopback-action Po10.10 drop
+```
+
+#### Show
+The following command can be used to show loopback action:
+```
+show ip interfaces loopback-action 
+```
+Example:
+```
+show ip interfaces loopback-action
+Interface     Action      
+------------  ----------  
+Ethernet248   drop     
+Vlan100       forward     
+PortChannel1  drop
+Ethernet0.10  drop
+Po10.10       drop
+```
+### Related DUT configuration files
+
+```
+"INTERFACE": {
+    "Ethernet248": {
+        "loopback_action": "drop"
+    },
+},
+"PORTCHANNEL_INTERFACE": {
+    "PortChannel1": {
+        "loopback_action": "drop"
+    },
+},
+"VLAN_INTERFACE": {
+    "Vlan100": {
+        "loopback_action": "drop"
+    },
+},
+"VLAN_SUB_INTERFACE": {
+    "Ethernet0.10": {
+        "admin_status": "up",
+        "loopback_action": "drop",
+        "vlan": "10"
+    },
+    "Po10.10": {
+        "loopback_action": "drop",
+        "admin_status": "up",
+        "vlan": "10"
+    }
+}
+```
+### Supported topology
+The test will be supported on any topology
+
+
+## Test cases
+
+### Test cases #1 - Verify the loopback action can be configured and worked as expect,the interface type will cover all the 5 types of ip interface.
+1. Verify the default action with sending traffic: 
+   - Verify the traffic will be forwarded as expected.
+   - The RIF TX_ERR counter will not increase as expected
+2. Configure the loopback action to drop on the interface with cli command
+3. Verify the the loopback action is configured correctly with show cli command.
+4. Send traffic to do the validation: 
+   - Verify the traffic will be dropped as expected.
+   - The RIF TX_ERR counter will increase as expected
+5. Change it back to forward
+6. Verify the the loopback action is configured correctly with show cli command.
+7. Send traffic to do the validation
+   - Verify the traffic will be forwarded as expected.
+   - The RIF TX_ERR counter will not increase.
+   
+### Test cases #2 - Disable/Enable the interface and check if the loopback action still work, the interface type will cover all types of ip interface except vlan interface.
+1. Configure the loopback action on the interface, some to drop, some to forward.
+2. Disable the interfaces
+3. Enable the interfaces back
+4. Verify the loopback action is not changed with show cli command.
+5. Send traffic to do the validation: 
+   - Verify the traffic will be dropped/forwarded as expected.
+   - The RIF TX_ERR counter will increase on the interface that the loppback action is configured to drop
+   - The RIF TX_ERR counter will not increase on the interface that the loppback action is configured to forward
+
+### Test cases #3 - Verify loopback action is saved after reboot(config reload reboot, fast-reboot), the interface type will cover all the 5 types of ip interface.
```

Comment:
We would need one test case specifically to cover warmboot

Reply:
add warmboot type